.github/workflows/changeset.yaml

```diff
@@ -23,3 +24,21 @@ jobs:
           commit: "version packages"
           version: yarn ci:version
           publish: yarn ci:publish
+      - name: Notify Slack (success)
+        if: steps.changesets.outputs.published == 'true'
+        uses: slackapi/slack-github-action@v1.24.0
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+        with:
+          channel-id: "wol_kaizen"
+          payload-file-path: "../payload/slack-notify-release.json"
+      - name: Notify Slack (failure)
+        if: steps.changesets.outputs.published == 'false'
+        uses: slackapi/slack-github-action@v1.24.0
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+        with:
+          channel-id: "wol_kaizen"
+          payload-file-path: "../payload/slack-notify-release.json"
```

Comment:
Since there's no difference between success and failure, doesn't it mean you can just get rid of the condition and just run the steps?

Reply:
Hmm I think the output value only exists when publish is run, so having the conditions here means that it won't notify us of a new Changeset PR.

Reply:
I wonder if we could check whether it's defined 🤔